### PR TITLE
Rename `createWebStorage` => `createSyncStorage`

### PR DESCRIPTION
**Pull request by todorone**

### CODE REVIEW

**Comment by TkDodo** [comment]
looks really good, thank you

File: docs/guides/migrating-to-react-query-4.md [hunk 1]
```diff
@@ -201,17 +201,17 @@ React.useEffect(() => mySideEffectHere(data), [data])
 
 ### `persistQueryClient` and the corresponding persister plugins are no longer experimental and have been renamed
 
-The plugins `createWebStoragePersistor` and `createAsyncStoragePersistor` have been renamed to [`createWebStoragePersister`](/plugins/createWebStoragePersister) and [`createAsyncStoragePersister`](/plugins/createAsyncStoragePersister) respectively. The interface `Persistor` in `persistQueryClient` has also been renamed to `Persister`. Checkout [this stackexchange](https://english.stackexchange.com/questions/206893/persister-or-persistor) for the motivation of this change.
+The plugins `createSyncStoragePersistor` and `createAsyncStoragePersistor` have been renamed to [`createSyncStoragePersister`](/plugins/createSyncStoragePersister) and [`createAsyncStoragePersister`](/plugins/createAsyncStoragePersister) respectively. The interface `Persistor` in `persistQueryClient` has also been renamed to `Persister`. Checkout [this stackexchange](https://english.stackexchange.com/questions/206893/persister-or-persistor) for the motivation of this change.
```
Comment (TkDodo) [inline docs/guides/migrating-to-react-query-4.md:204]: I think we have to keep the reference to the old name here :)

```suggestion
The plugins `createWebStoragePersistor` and `createAsyncStoragePersistor` have been renamed to [`createSyncStoragePersister`](/plugins/createSyncStoragePersister) and [`createAsyncStoragePersister`](/plugins/createAsyncStoragePersister) respectively. The interface `Persistor` in `persistQueryClient` has also been renamed to `Persister`. Checkout [this stackexchange](https://english.stackexchange.com/questions/206893/persister-or-persistor) for the motivation of this change.
```

Comment (todorone) [reply docs/guides/migrating-to-react-query-4.md]: Oops, thanks :)

**Comment by TkDodo** [comment]
we have incorporated the change into the re-brand PR that will land shortly. Thanks 🙏

**Pull request closed without merging**